generated/typespec-ts/src/api/operations.ts

```diff
@@ -0,0 +1,1708 @@
+  CreateFineTune200Response,
+  CreateFineTuneDefaultResponse,
+  CreateFineTuningJob200Response,
+  CreateFineTuningJobDefaultResponse,
+  CreateImage200Response,
+  CreateImageDefaultResponse,
+  CreateImageEdit200Response,
+  CreateImageEditDefaultResponse,
+  CreateImageVariation200Response,
+  CreateImageVariationDefaultResponse,
+  CreateModeration200Response,
+  CreateModerationDefaultResponse,
+  CreateTranscription200Response,
+  CreateTranscriptionDefaultResponse,
+  CreateTranslation200Response,
+  CreateTranslationDefaultResponse,
+  DeleteFile200Response,
+  DeleteFileDefaultResponse,
+  DeleteOperation200Response,
+  DeleteOperationDefaultResponse,
+  DownloadFile200Response,
+  DownloadFileDefaultResponse,
+  isUnexpected,
+  ListFiles200Response,
+  ListFilesDefaultResponse,
+  ListFineTuneEvents200Response,
+  ListFineTuneEventsDefaultResponse,
+  ListFineTunes200Response,
+  ListFineTunesDefaultResponse,
+  ListFineTuningEvents200Response,
+  ListFineTuningEventsDefaultResponse,
+  ListModels200Response,
+  ListModelsDefaultResponse,
+  ListPaginatedFineTuningJobs200Response,
+  ListPaginatedFineTuningJobsDefaultResponse,
+  OpenAIContext as Client,
+  Retrieve200Response,
+  RetrieveDefaultResponse,
+  RetrieveFile200Response,
+  RetrieveFileDefaultResponse,
+  RetrieveFineTune200Response,
+  RetrieveFineTuneDefaultResponse,
+  RetrieveFineTuningJob200Response,
+  RetrieveFineTuningJobDefaultResponse,
+} from "../rest/index.js";
+import {
+  StreamableMethod,
+  operationOptionsToRequestParameters,
+} from "@azure-rest/core-client";
+import { uint8ArrayToString, stringToUint8Array } from "@azure/core-util";
+import {
+  CreateTranscriptionOptions,
+  CreateTranslationOptions,
+  CreateChatCompletionOptions,
+  CreateFineTuningJobOptions,
+  ListPaginatedFineTuningJobsOptions,
+  RetrieveFineTuningJobOptions,
+  ListFineTuningEventsOptions,
+  CancelFineTuningJobOptions,
+  CreateCompletionOptions,
+  CreateEditOptions,
+  CreateEmbeddingOptions,
+  ListFilesOptions,
+  CreateFileOptions,
+  RetrieveFileOptions,
+  DeleteFileOptions,
+  DownloadFileOptions,
+  CreateFineTuneOptions,
+  ListFineTunesOptions,
+  RetrieveFineTuneOptions,
+  ListFineTuneEventsOptions,
+  CancelFineTuneOptions,
+  ListModelsOptions,
+  RetrieveOptions,
+  DeleteOptions,
+  CreateImageOptions,
+  CreateImageEditOptions,
+  CreateImageVariationOptions,
+  CreateModerationOptions,
+} from "../models/options.js";
+
+export function _createTranscriptionSend(
+  context: Client,
+  audio: CreateTranscriptionRequest,
+  options: CreateTranscriptionOptions = { requestOptions: {} }
+): StreamableMethod<
+  CreateTranscription200Response | CreateTranscriptionDefaultResponse
+> {
+  return context
+    .path("/audio/transcriptions")
+    .post({
+      ...operationOptionsToRequestParameters(options),
+      contentType: (options.contentType as any) ?? "multipart/form-data",
+      body: {
+        file: uint8ArrayToString(audio["file"], "binary"),
```

Comment:
[Compile Error]
src/api/operations.ts:148:49 - error TS2345: Argument of type '"binary"' is not assignable to parameter of type 'EncodingType'.

148         file: uint8ArrayToString(audio["file"], "binary")